<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>standEdgeWdg</class>
 <widget class="QWidget" name="standEdgeWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>standEdgeWdg</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" rowspan="2" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>属性</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <property name="spacing">
         <number>3</number>
        </property>
        <item row="0" column="1">
         <widget class="QPushButton" name="bt_selectRoi">
          <property name="text">
           <string>选择区域</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="bt_setOrigin">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>设入中心</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QCheckBox" name="cb_detectEndPoint">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>检测端点：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>最大角度：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sp_maxAngle">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="maximum">
             <number>45</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="bt_helpRect">
          <property name="text">
           <string>辅助选框</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>上边</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_9"/>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>下边</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_10"/>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>左边</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_11"/>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>右边</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_12"/>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>1</number>
           </property>
           <property name="bottomMargin">
            <number>1</number>
           </property>
           <property name="spacing">
            <number>1</number>
           </property>
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_9">
             <property name="title">
              <string/>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="label_25">
                <property name="text">
                 <string>灰度差异:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="sp_minGrayDiff">
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::UpDownArrows</enum>
                </property>
                <property name="minimum">
                 <number>5</number>
                </property>
                <property name="maximum">
                 <number>200</number>
                </property>
               </widget>
              </item>
              <item row="4" column="0" colspan="4">
               <widget class="QGroupBox" name="grp_high">
                <property name="title">
                 <string/>
                </property>
                <layout class="QGridLayout" name="gridLayout">
                 <item row="2" column="3">
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>偏离修正:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="4" colspan="3">
                  <widget class="QSpinBox" name="sp_offsetPlus">
                   <property name="minimum">
                    <number>-99</number>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1" colspan="2">
                  <widget class="QSpinBox" name="sp_searchLineGap">
                   <property name="buttonSymbols">
                    <enum>QAbstractSpinBox::UpDownArrows</enum>
                   </property>
                   <property name="minimum">
                    <number>1</number>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1" colspan="2">
                  <widget class="QSpinBox" name="sp_paichu">
                   <property name="buttonSymbols">
                    <enum>QAbstractSpinBox::UpDownArrows</enum>
                   </property>
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>50</number>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_22">
                   <property name="layoutDirection">
                    <enum>Qt::LeftToRight</enum>
                   </property>
                   <property name="text">
                    <string> 扫描间距：</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QCheckBox" name="cb_paichu">
                   <property name="toolTip">
                    <string>启用后将会对偏移一定距离的异常边界线进行排除</string>
                   </property>
                   <property name="layoutDirection">
                    <enum>Qt::RightToLeft</enum>
                   </property>
                   <property name="text">
                    <string>启用排除线: </string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1" colspan="2">
                  <widget class="QSpinBox" name="sp_lineOffset">
                   <property name="buttonSymbols">
                    <enum>QAbstractSpinBox::UpDownArrows</enum>
                   </property>
                   <property name="minimum">
                    <number>-99</number>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_24">
                   <property name="text">
                    <string> 边线偏移：</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="3">
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>分割阈值：</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="3">
                  <widget class="QLabel" name="label_15">
                   <property name="text">
                    <string>比例排除：</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="3">
                  <widget class="QLabel" name="label_14">
                   <property name="text">
                    <string>拟合系数：</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="4" colspan="3">
                  <widget class="QDoubleSpinBox" name="dsp_scalePaichu">
                   <property name="buttonSymbols">
                    <enum>QAbstractSpinBox::UpDownArrows</enum>
                   </property>
                   <property name="maximum">
                    <double>1.000000000000000</double>
                   </property>
                   <property name="singleStep">
                    <double>0.100000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="4" colspan="3">
                  <widget class="QDoubleSpinBox" name="dsp_factor">
                   <property name="decimals">
                    <number>1</number>
                   </property>
                   <property name="minimum">
                    <double>1.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>3.000000000000000</double>
                   </property>
                   <property name="singleStep">
                    <double>0.500000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="4" colspan="3">
                  <widget class="QSpinBox" name="sp_grayval">
                   <property name="buttonSymbols">
                    <enum>QAbstractSpinBox::UpDownArrows</enum>
                   </property>
                   <property name="minimum">
                    <number>5</number>
                   </property>
                   <property name="maximum">
                    <number>255</number>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="QCheckBox" name="cb_leadSize">
                   <property name="layoutDirection">
                    <enum>Qt::RightToLeft</enum>
                   </property>
                   <property name="text">
                    <string>管脚大小筛选</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="1" colspan="2">
                  <widget class="QSpinBox" name="sp_leadSize">
                   <property name="buttonSymbols">
                    <enum>QAbstractSpinBox::UpDownArrows</enum>
                   </property>
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>1000</number>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="0">
                  <widget class="QCheckBox" name="cb_outlier">
                   <property name="layoutDirection">
                    <enum>Qt::RightToLeft</enum>
                   </property>
                   <property name="text">
                    <string>去除离群点  </string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="bt_highLevel">
                <property name="text">
                 <string>高级设置</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_53">
                <property name="text">
                 <string>端点扫描灰度：</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QSpinBox" name="sp_edgeGray">
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::UpDownArrows</enum>
                </property>
                <property name="minimum">
                 <number>5</number>
                </property>
                <property name="maximum">
                 <number>200</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="title">
        <string>定位误差范围</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <property name="spacing">
         <number>3</number>
        </property>
        <item row="1" column="5">
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>107</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="5">
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>107</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="sp_errHeightMin">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="sp_errAngle">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>45</number>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QSpinBox" name="sp_errCenterOffset">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="cb_errHeight">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>高度：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="cb_errAngle">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>角度：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="sp_errWidthMin">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="cb_errWidth">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>宽度：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QSpinBox" name="sp_errHeightMax">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QSpinBox" name="sp_errWidthMax">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QCheckBox" name="cb_errCenterOffset">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>中心偏移：</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer_17">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>396</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
